apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/GrunnlagsendringshendelseDao.kt

```diff
@@ -107,6 +107,23 @@ class GrunnlagsendringshendelseDao(val connection: () -> Connection) {
         }
     }
 
+    fun aapneGrunnlagsendringshendelserForBehandlingId(behandlingId: UUID) {
+        with(connection()) {
+            prepareStatement(
+                """
+                    UPDATE grunnlagsendringshendelse
+                    SET kommentar = null,
+                        status = ?
+                    WHERE behandling_id = ?
+                """.trimIndent()
+            ).use {
+                it.setString(1, GrunnlagsendringStatus.SJEKKET_AV_JOBB.toString())
```

Comment:
Det er noe som føles rart med å "hardkode" en SQL-parameter. Her definerer vi en SQL, som tilsier at status kan være hva som helst, men så blir den alltid kalt med samme verdi. Kan vel ikke si at det hadde sett bedre ut med interpolation, men kanskje om det var dratt ut i en egen metode selv om det praktisk sett ville vært identisk...

Reply:
Ja, er forsåvidt enig her. Ville bare unngå å skrive inn definisjonen i teksten, men skal se om det blir mer naturlig med en funksjon for det.

Reply:
Jeg synes det ble litt mer herk med funksjon, og vi følger dette patternet en del andre steder. Endte opp med å være konsekvent med tidligere, og så kan det evt. endres til noe likt senere.